site/app/templates/generic/Popup.twig

```diff
@@ -10,6 +10,7 @@
                         {% block title_tag %}
                             <h1>{% block title %}Untitled Form{% endblock %}</h1>
                             <button
+                                data-close="close-button"
```

Comment:
See https://docs.cypress.io/guides/references/best-practices#Selecting-Elements
For best use with Cypress, we should use `data-cy, data-test, or data-testid`. I am partial to `data-testid` because it is more descriptive to what it is.

Reply:
alright, will replace `data-close` with `data-testid`